src/components/Animal/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import './style.css';
+//nezapomínej změnit class na className
```

Comment:
editory či jiné nástroje (eslint) se dají pak nastavit na to aby ti takovéhle věci hlídaly :)